src/schemas.py

```
@@ -1,7 +1,7 @@
 import datetime
 from typing import Annotated, Optional
 
-from pydantic import BaseModel, ConfigDict, Field, field_validator
+from pydantic import BaseModel, ConfigDict, Field, computed_field, field_validator
```

Ruff (0.11.9)

`pydantic.computed_field` imported but unused

Remove unused import: `pydantic.computed_field`

(F401)